grepos/app/src/main/java/net/boondockradio/grepos/dto/Owner.java

```diff
@@ -0,0 +1,367 @@
+
+package net.boondockradio.grepos.dto;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Owner {
+
+    @SerializedName("login")
+    @Expose
+    private String login;
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("avatar_url")
+    @Expose
+    private String avatarUrl;
+    @SerializedName("gravatar_id")
+    @Expose
+    private String gravatarId;
+    @SerializedName("url")
+    @Expose
+    private String url;
+    @SerializedName("html_url")
+    @Expose
+    private String htmlUrl;
+    @SerializedName("followers_url")
+    @Expose
+    private String followersUrl;
+    @SerializedName("following_url")
+    @Expose
+    private String followingUrl;
+    @SerializedName("gists_url")
+    @Expose
+    private String gistsUrl;
+    @SerializedName("starred_url")
+    @Expose
+    private String starredUrl;
+    @SerializedName("subscriptions_url")
+    @Expose
+    private String subscriptionsUrl;
+    @SerializedName("organizations_url")
+    @Expose
+    private String organizationsUrl;
+    @SerializedName("repos_url")
+    @Expose
+    private String reposUrl;
+    @SerializedName("events_url")
+    @Expose
+    private String eventsUrl;
+    @SerializedName("received_events_url")
+    @Expose
+    private String receivedEventsUrl;
+    @SerializedName("type")
+    @Expose
+    private String type;
+    @SerializedName("site_admin")
+    @Expose
+    private Boolean siteAdmin;
```

Comment:
Its a small thing. Not important here.
But `Boolean` and `boolean` are different things.
`boolean` is a primitive.
`Boolean` is an object.

So unless we have a reason to use a `Boolean` object we should use `boolean`

Same for `Integer/int`, `Double/double` etc